src/civis/resources/_resources.py

```diff
@@ -22,8 +23,6 @@
     os.path.dirname(os.path.realpath(__file__)),
     "civis_api_spec.json",
 )
-with open(API_SPEC_PATH) as f:
-    API_SPEC = json.load(f, object_pairs_hook=OrderedDict)
```

Comment:
Having to load a file for JSON parsing at the module level incurs quite a bit of import overhead. `API_SPEC` is only used in the test suite, so I'm moving it there.